assets/sass/protocol/includes/_mixins.scss

```diff
@@ -168,43 +166,41 @@
     white-space: nowrap;
 }
 
-
 // Flip an element horizontally for RTL locales such as Arabic and Hebrew.
 // Usually applied to pseudo-elements with a background-image, not for text.
 @mixin flip-horizontally {
     @include transform(scale(-1, 1));
 }
 
-
 // Adds a right-pointing arrow after
 @mixin trailing-arrow {
-    &:after {
-        content: "\00A0\25B8"; // space+triangle
+    &::after {
+        content: '\00A0\25B8'; // space+triangle
         white-space: nowrap;
     }
 }
 
 // Adds a left-pointing arrow before
 @mixin leading-arrow {
-    &:before {
-        content: "\25B8\00A0"; // triangle+space
+    &::before {
+        content: '\25B8\00A0'; // triangle+space
         white-space: nowrap;
+
         @include transform(rotate(180deg));
     }
 }
 
-
 // Multiple columns
 @mixin multi-column($width: auto, $count: auto, $gap: normal) {
-    -webkit-column-width: $width;
-       -moz-column-width: $width;
-            column-width: $width;
-    -webkit-column-count: $count;
-       -moz-column-count: $count;
-            column-count: $count;
-      -webkit-column-gap: $gap;
-         -moz-column-gap: $gap;
-              column-gap: $gap;
+    column-width: $width;
+    column-width: $width;
+    column-width: $width;
+    column-count: $count;
+    column-count: $count;
+    column-count: $count;
+    column-gap: $gap;
+    column-gap: $gap;
+    column-gap: $gap;
```

Comment:
Nit: looks like the linter created some duplicate properties here, can these now be trimmed down?